app/aggregates/reviewing/commands/receive_application.rb

```diff
@@ -4,15 +4,16 @@ class ReceiveApplication < Command
     attribute :submitted_at, Types::DateTime
     attribute? :parent_id, Types::Uuid.optional
     attribute :work_stream, Types::WorkStreamType
+    attribute? :application_type, Types::ApplicationType.optional
     attribute? :correlation_id, Types::Uuid.optional
     attribute? :causation_id, Types::Uuid.optional
 
     def call
       with_review do |review|
-        review.receive_application(submitted_at:, parent_id:, work_stream:)
+        review.receive_application(submitted_at:, application_type:, parent_id:, work_stream:)
       end
 
-      return unless parent_id
+      return unless supersedes?
```

Comment:
As pse applications are technically resubmissions, here we make sure only actual resubmitted applications get the supersede event